### ISSUE TITLE
Unicode symbols for locale files

### ISSUE THREAD

**Issue by imDaniX** [mcMMO-Dev/mcMMO#4269]
I guess that's in the title. Right now it's just horrifying to work with.

**Comment by Quavelen** [comment]
Unfortunately @imDaniX, this is something on Java's end.

> 23/06/2019 - 19:30
> nossr50: 
> we use a Locale library built into Java, and in Java 8 they did not support unicode

You'll need to use a unicode to UTF16 converter; there are many online which are simply copy and paste.
For example: https://onlineunicodetools.com/convert-unicode-to-utf16

**Comment by imDaniX** [comment]
EssX is using it too, but they overcame it - https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/com/earth2me/essentials/I18n.java

**Comment by Quavelen** [comment]
I've spoken to @nossr50 about this @imDaniX, and he explained that whilst the library we use was indeed limited during Java 8, the library was updated to support Unicode properly, in Java 9. So this is actually feasible, but since there are online converters which make it very easy to convert to UTF-16, and nossr50 has more important things to work on, this is a low priority. 


However, I've reopened this issue and added the 'Feature Request' label. When nossr50 has a lighter workload, he'll be able to take a look at requests like this one.
